### PR TITLE
[TRANSLATION] Terms of use added to en.json for translations 

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**
Terms of Use need to be translated for an upcoming feature. The English text has been added to the en.json for translation purposes.

**Screenshots/Recordings**
NA

**Issue**
Progresses MetaMask/mobile-planning#466 

**Checklist**
* [NA] There is a related GitHub issue
* [NA] Tests are included if applicable
* [NA] Any added code is fully documented

### CODE REVIEW

File: locales/languages/en.json [hunk 1]
```diff
@@ -2194,5 +2194,74 @@
...
+    "section_12_7": "12.7 Language. All communications and notices made or given pursuant to this Agreement must be in the English language. If we provide a translation of the English language version of this Agreement, the English language version of the Agreement will control if there is any conflict.",
+    "section_12_8": "12.8 Notice.\n\n(a) To You. We may provide any notice to you under this Agreement using commercially reasonable means, including: (i) posting a notice on the Site; (ii) sending a message to the email address then associated with your account; or (iii) using public communication channels . Notices we provide by posting on the Site or using public communication channels will be effective upon posting, and notices we provide by email will be effective when we send the email. It is your responsibility to keep your email address current to the extent you have an account. You will be deemed to have received any email sent to the email address then associated with your account when we send the email, whether or not you actually receive the email.\n\n(b) To Us. To give us notice under this Agreement, you must contact us by email at notices@consensys.net.",
```
Comment (tommasini) [inline locales/languages/en.json:2259]: It seems to me that it's missing a d on receive: whether or not you actually `received`

Comment (tommasini) [inline locales/languages/en.json:2259]: Between channels and the final dot, there is an empty space, don't know if it was intended -> `the email communication channels .`

Comment (sethkfman) [reply locales/languages/en.json]: Removed

Comment (sethkfman) [reply locales/languages/en.json]: I did not make this change since it was drafted by Legal.

**Review by tommasini — APPROVED** [review]
LGTM

**Review by gantunesr — APPROVED** [review]

**Pull request merged**